### ISSUE TITLE
Bot's showing offline.

### ISSUE THREAD

**Issue by Irresplendence** [Darkside138/DiscordSoundboard#139]
Howdy Howdy,

I'd taken the bot down a moment to do some maintenance to something, and tried to boot her back up. Afterwards, the bot just shows offline. I thought that was odd, and tried changing some stuff around, then ultimately, went through the initial setup process, i.e inviting the bot again, so on and so forth. Nothing still, and nothing that seems to reflect any issues within the boot process, and I'm not quite sure where to go from here. : ^)

**Comment by Irresplendence** [comment]
So, slight update. Tried running that batch file again and it actually auto-closes. I'm pretty sure Java and everything are up to date, but I'll double check just to be sure.

**Comment by jusplainmike** [comment]
Can you provide some more info like operating system, java version, and any log output from bot.

**Comment by Irresplendence** [comment]
I'm sorry, I actually did some tinkering with the batch file and added some pauses to see what was the issue and never updated. I think my initial problem was it just slammed itself shut without me being able to see what the issue was. Turns out I'd forgotten to add login creds for the database and without them it just closed. Added them and it seems to be working within normal capacity. Sorry about that.
